tinyagentos/routes/skill_exec.py

```
@@ -482,6 +511,7 @@ async def execute_skill(skill_id: str, request: Request):
 
     try:
         result = await impl(args, request)
+        _capture_tool_receipt(request, skill_id, args, result)
         return JSONResponse(result)
     except Exception as exc:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)